src/vue/modules/fees/store/index.spec.js

```diff
@@ -0,0 +1,103 @@
+import { mutations, getters } from './index'
+import { types } from './types'
+import { filterFees } from '../helpers/filter-fees'
+
+describe('fees.module', () => {
+  const accountId = 'GDIU5OQPAFPNBP75FQKMJTWSUKHTQTBTHXZWIZQR4DG4QRVJFPML6TTJ'
+  const accountRoleId = '5'
+  const fees = [{
+    account: {
+      id: accountId,
+    },
+    fixed: '1.000000',
+  }, {
+    accountRole: {
+      id: accountRoleId,
+    },
+    fixed: '5.000000',
+  }, {
+    fixed: '7.000000',
+  }]
+
+  describe('mutations', () => {
+    let state
+
+    beforeEach(() => {
+      state = {
+        accountId: '',
+        accountRoleId: '',
+        fees: [],
+      }
+    })
+
+    it('SET_ACCOUNT_ID should properly modify state', () => {
+      mutations[types.SET_ACCOUNT_ID](state, accountId)
+
+      expect(state).to.deep.equal({
+        accountId: accountId,
+        accountRoleId: '',
+        fees: [],
+      })
+    })
+
+    it('SET_ACCOUNT_ROLE_ID should properly modify state', () => {
+      mutations[types.SET_ACCOUNT_ROLE_ID](state, accountRoleId)
+
+      expect(state).to.deep.equal({
+        accountId: '',
+        accountRoleId: accountRoleId,
+        fees: [],
+      })
+    })
+
+    it('SET_FEES should properly modify state', () => {
+      mutations[types.SET_ACCOUNT_ID](state, accountId)
+      mutations[types.SET_ACCOUNT_ROLE_ID](state, accountRoleId)
```

Comment:
this test looks bad because you have a lot of logic you don't test here - you call other mutations you don't test, and expecting state to have properties that are set by this mutations. Having hacky tests is a straight sign of having bad code

Reply:
I explained you the problems it the module code above

Reply:
Updated